igvc_navigation/launch/localization.launch

```diff
@@ -21,7 +21,7 @@
 
       <!-- for if the IMU (after magnetic declination adjustment)
            does not read 0 -->
-      <param name="yaw_offset" value="1.57"/>
+      <param name="yaw_offset" value="0"/>
```

Comment:
We should double check this on the robot. This may be a on the robot configuration. In that case we should rotate our IMU in the URDF to match whatever is on the robot. 

TLDR; check IMU orientation IRL and make sure simulation matches not the other way around.

Reply:
So I talked to justin about this yesterday. He mentioned we no longer need the 'yaw_offset' parameter. The reason it was needed at competition was because the IMU wasn't given enough time to converge on an accurate yaw value, so it set its initial yaw to an unfiltered, inaccurate value - this required manual correction via the yaw_offset parameter.

Reply:
Then I would remove the parameter entirely, but it was mostly used for actual offset.

Reply:
o.k. i'll get rid of it. I'll still include it as a parameter in the documentation just for reference.